liiweb/views/legislation.py

```diff
@@ -47,6 +47,19 @@ def get_context_data(self, **kwargs):
 
         context["legislation_table"] = LegislationSerializer(qs, many=True).data
 
+        site_jurisdictions = pj_settings().document_jurisdictions.all()
+        if site_jurisdictions.count() == 1:
+            jurisdiction_profile = JurisdictionProfile.objects.filter(
+                jurisdiction=site_jurisdictions.first()
+            ).first()
+            if jurisdiction_profile:
+                context[
+                    "jurisdiction_entity_profile"
```

Comment:
Name these `entity_profile` and `entity_profile_title` and then the template doesn't needto change.